CHANGELOG.md

```diff
@@ -1891,34 +1947,35 @@ You can control what will be generated and how with corresponding configuration
 
 Under dfx.json → "canisters" → "<canister_name>", developers can add a "declarations" config. Options are:
 
-* "output" → directory to place declarations for that canister | default is "src/declarations/<canister_name>"
+- "output" → directory to place declarations for that canister | default is "src/declarations/<canister_name>"
 
-* "bindings" → [] list of options, ("js", "ts", "did", "mo") | default is "js", "ts", "did"
+- "bindings" → [] list of options, ("js", "ts", "did", "mo") | default is "js", "ts", "did"
```

Comment:
This mass reformatting of CHANGELOG.md is nice, but would have been better done as a separate PR, in order to make it easier to see what differences there are related to the changes in this PR.

Reply:
Reverted